gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/ImportGradesHelper.java

```diff
@@ -52,8 +52,8 @@
 	public final static int USER_NAME_POS = 1;
 
 	// patterns for detecting column headers and their types
-	final static Pattern ASSIGNMENT_WITH_POINTS_PATTERN = Pattern.compile("([\\w ]+ \\[[0-9]+(\\.[0-9][0-9]?)?\\])");
-	final static Pattern ASSIGNMENT_COMMENT_PATTERN = Pattern.compile("(\\* [\\w ]+)");
+	final static Pattern ASSIGNMENT_WITH_POINTS_PATTERN = Pattern.compile("(\.\* \\[[0-9]+(\\.[0-9][0-9]?)?\\])");
```

Comment:
I want the regex to say accept everything (.*) so I thought it was just one backslash because those are both special characters.

The previous regex was asking for any word ("\w") so they have to escape the backslash so it becomes ("\\w")